output-tests/README.md

```diff
@@ -0,0 +1,26 @@
+These tests are intended to validate that implementations are correctly generating output in accordance with the specification.
+
+Output was initially specified with draft 2019-09.  It remained unchanged for draft 2020-12 <!-- do we need explicit tests for 2002-12? -->, but will receive an update with the next release.
+
+## Organization
+
+The tests are organized into two categories: content and structure.
+
+Content tests verify that the keywords are producing the correct annotations.  Since there are no requirements on the content of error messages, there's not much that can be verified for them, if they're even generated.  These tests need to extensively cover the annotation behaviors of each keyword.  The expected output format for these tests is `basic`.
+
+Structure tests verify that the structures of the various formats (i.e. `flag`, `basic`, `detailed`, `verbose`) are correct.  These tests don't need to cover each keyword; rather they need to sufficiently cover the various aspects of building the output structures by using whatever keywords are necessary to do so.
+
+## Test Files
+
+The content of a test file is the same as the validation tests in `tests/`, however an `output` property has been added to each test case.
```

Comment:
Probably a decent idea to regardless copy the `test-schema.json` file here and make sure it matches these (and add a test to CI that tests are valid under it in this directory -- which I think should just be some copy pasting).

Reply:
I don't know if we need to copy it if it's in the same repo.

One file that I _would_ like to copy is the output schema (that's now updated to the `draft-next` version).  I think referencing that would reduce a lot of boilerplate.